meetings/2022-03-14.md

```diff
@@ -0,0 +1,108 @@
+* ...: You would disclose some kind of secret allowing verifying the issuer. So pretty much just public key of the issuer.
+* NAS: So pretty much user would be providing OP rather than WebID.
+* MB: I mean that OP would keep private key for each of the user. And as a user I would disclosed the public key associated with private key used by OP for my WebID.
+* NAS: My understanding was that we want to center things around WebID. But here user would need to provide OP to the client.
+* AC: It sounds like we can use `did:key` or something described in WebID-TLS. Is it separate from Solid-OIDC? We could have different kinds of interaction which could look more like WebID-TLS. This is valuable use case but not everyone use case.
+* eP: Currently in [section 8.1](https://solid.github.io/solid-oidc/#id-token-validation), we say "unless the verifying party accepts keys via other means...". IOW, we already have this described "via other means", so could we use this to extend Solid-OIDC? Without disclosing issuer? I.e., just like Solid-OIDC but where the keys are retrieved via some other means.
+* eP: It would be useful if the keys were also in JWK, that would keep the feature very similar to the current flow
+* MB: I don't have well constructed opinion if this is too much of a departure to have it there. If it is compatible it would be nice to have it in Solid-OIDC.
+* NAS: Currently it is possible, when you log user in based on WebID you know the WebID and discover OP from there. Then you can get OP keys. Is it possible that you can get token which was spoofed? There is no strong relationship between OP and the key. This is a strong change compared to Solid-OIDC. That may warrant to have it separate. 
+* AC: Let's say you have 2 storage servers A and B. With A there is this pre-existing trust where it trusts issuer P. Server B does not have the same trust with issuer P. What happens if a user tries to log in to server A with issuer P, they can do it fine. They would fail to log in to issuer B since they don't have that trust.
+* ...: What might be interesting as an alternative, is rather than trying to say I trust this particular issuer or specific issuers. I trust a [federation of issuer](https://openid.net/specs/openid-connect-federation-1_0.html). If you trust federation of let's say 20 or 30. It will not give away that much information. This may answer some of the use cases. We would have the question how you discover the federation.
+* MB: The federation would still be a group of issuers adhering to certain set of values. It may be more private, but still disclose some association with the values carried by such federation.
+* eP: Aaron, is it related to issuer restriction case? For example `acp:issuer`
+* AC: The first thing is AS looks up the WebID profile to check the issuer is trusted by this WebID profile. Because I can write a valid ID TOken claiming to be anyone. How can an AS know that this particular ID Token is to be trusted.
```

Comment:
```suggestion
* AC: The first thing is [AS looks up the WebID profile](https://solid.github.io/solid-oidc/#basic-flow) to check the issuer is trusted by this WebID profile. Because I can write a valid ID Token claiming to be anyone. How can an AS know that this particular ID Token is to be trusted.
```